chia/_tests/blockchain/test_blockchain.py

```diff
@@ -2606,17 +2610,18 @@ async def test_invalid_transactions_ref_list(
         await _validate_and_add_block(b, blocks[-1])
         assert blocks[-1].transactions_generator is not None
 
-        blocks = bt.get_consecutive_blocks(
-            1,
-            block_list_input=blocks,
-            guarantee_transaction_block=True,
-            transaction_data=tx,
-            block_refs=[blocks[-1].height],
-        )
-        block = blocks[-1]
-        # once the hard fork activated, we no longer use this form of block
-        # compression anymore
-        assert len(block.transactions_generator_ref_list) == 0
+        if consensus_mode < ConsensusMode.HARD_FORK_3_0:
```

Comment:
add comment to explain that this test won't work after the hard fork, because block refs